front/components/assistant/conversation/actions/VisualizationActionIframe.tsx

```diff
@@ -153,7 +153,7 @@ export function VisualizationActionIframe({
   return (
     <div className="relative flex flex-col">
       {showSpinner && (
-        <div className="absolute inset-0 z-50 flex items-center justify-center bg-white">
+        <div className="absolute inset-0 flex items-center justify-center bg-white">
```

Comment:
I simply removed the z-index, do you remember why you added it @spolu ? To make sure I'm not introducing another regression 🙏🏻

Reply:
huh likely debugging. I think it's fine to remove